examples/single-page-app/ui/src/components/Users.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  Flex,
+  Image,
+  Link,
+  Td,
+  Th,
+} from '@chakra-ui/react'
+import React from "react"
+
+import {
+  IRepoInfo,
+  IUserLogin} from "../@types/app"
+
+
+export const RelatedUserTableHeaders = () => {
```

Comment:
Is this first use of Typescript in Envoy-proper? Do we need a style guide / formatter?

Reply:
only use im aware of - we have some node stuff already - but not react or ts i think

Reply:
re linter - its run in the verify.sh script for ts - its a bit hidden - i will move to envoy.code.check at some point - but for now its testing something else also

Reply:
Should we adopt https://google.github.io/styleguide/jsguide.html and configure linters for this? Or some opinionated guidance?

For small isolate scripts here and there it doesn't matter, but if we anticipate things growing over time, avoiding disagreeing on the little things when there is > 1 developer is helped by picking the arbitrary choices (plus various industry standard style guides usually have good opinions for the most part informed by lots of history).

Reply:
please lets kick this down the road for now - js linters are non-trivial - if can easily break things

i personally really do not want to adopt googles style guide (it took me long enough unpicking the non-standard idiosyncratic python as many other projects have had to do)

i would really rather this didnt block a relatively small amount of code being added - im not adverse to looking into it after

Reply:
OK, this one is fine.

Reply:
ticket added https://github.com/envoyproxy/envoy/issues/31665